packages/grub/efi.cfg

```diff
@@ -1,3 +1,5 @@
+set no_modules=y
+terminal_output console
```

Comment:
Is it possible to have both EFI and BIOS output to `serial`, or are `serial` and `console` the same thing?

Reply:
I asked BenH about this and will try to summarize his answer. 😁 

EFI and BIOS both support "console" and "serial" output.

For EFI, `console` is an API that's used to interact with the underlying hardware. We could try to bypass it by using `serial`, but that's not a great idea because EFI "owns" the device at this point.

For BIOS, `console` involves using BIOS interrupt calls for output. This can work but is inconsistent; for example, we don't get output on EC2 metal instances.

Using `serial` for BIOS lets us avoid the inconsistency on the platforms we support today, but might be the wrong choice on other platforms in the future if we run into something that needs `gfxterm` instead.

Using `console` for EFI should always be the correct choice.